sdk/python/feast/repo_operations.py

```diff
@@ -128,9 +128,22 @@ def parse_repo(repo_root: Path) -> ParsedRepo:
 
 
 @log_exceptions_and_usage
-def apply_total(repo_config: RepoConfig, repo_path: Path, skip_source_validation: bool):
-    from colorama import Fore, Style
-
+def plan_total(
+    repo_config: RepoConfig, repo_path: Path, skip_source_validation: bool
+) -> Tuple[
+    List[
+        Union[
+            BaseFeatureView, OnDemandFeatureView, Entity, FeatureService, FeatureTable,
+        ],
+    ],
+    List[
+        Union[
+            BaseFeatureView, OnDemandFeatureView, Entity, FeatureService, FeatureTable,
+        ]
+    ],
+    List[Union[FeatureView, FeatureTable]],
+    List[Union[FeatureView, FeatureTable]],
+]:
```

Comment:
nit - in my opinion this should be part of the feature store, so that you can run `feast plan` programatically. 

I think we shouldn't have any functionality that is only usable via the CLI, but not via the programatic API.

Reply:
Agree